### ISSUE TITLE
Could you please share any Vardial 2017 dataset url?

### ISSUE THREAD

**Issue by harrysimply** [aitor-mir/ADI#1]
Hi author,

I just read your paper on Interspeech 2020 which arise a new pattern to improve the performance on ADI task. It is a masterpiece and is helpful for I am working on.

The dataset the paper mentioned in Vardial 2017 which should be publican downloaded but I can not find the dataset when I search on Google. Could you please share the Vardial 2017 dataset download url?

Thanks!

**Comment by aitor-alvarez** [comment]
Hello Harry,

Thank you for your interest in this work. The VarDial data can be found here: https://github.com/aitor-arronte/dialectID/tree/master/data more specifically you can find the training set here: https://raw.githubusercontent.com/aitor-arronte/dialectID/master/data/train.vardial2017/wav.lst and the test set here: https://github.com/aitor-arronte/dialectID/blob/master/data/dev.vardial2017/wav.lst

As you can see those are lists of wav files that can be easily downloaded by making requests in a simple script. I hope this helps.

Best,
Aitor

**Comment by harrysimply** [comment]
Bravo!

Thanks Aitor! Your reply helps a lot! 

Regards,
Harry
